### PR TITLE
Replaced /main/ from /master/

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
Signed-off-by: Dishank Tiwari <dtiwari@progress.com>

## Description

Update /master/ to /main/ across the documentation.

## Checklist:
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Review by clintoncwolfe — APPROVED** [review]
Thanks!

File: docs-chef-io/content/inspec/resources/aws_ecr_repository.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ platform = "aws"
     parent = "inspec/resources/aws"
 +++
 
-[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/master/docs-chef-io/content/aws_ecr_repository.md)
+[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/main/docs-chef-io/content/aws_ecr_repository.md)
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_ecr_repository.md]: We can delete this entire line. This line just duplicates a Hugo function that does the same thing.
```suggestion
```

File: docs-chef-io/content/inspec/resources/aws_efs_file_system.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ platform = "aws"
     parent = "inspec/resources/aws"
 +++
 
-[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/master/docs-chef-io/content/inspec/rources/aws_efs_file_system.md)
+[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/main/docs-chef-io/content/inspec/rources/aws_efs_file_system.md)
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_efs_file_system.md]: Same as above.
```suggestion
```

File: docs-chef-io/content/inspec/resources/aws_elasticache_cluster.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ platform = "aws"
     parent = "inspec/resources/aws"
 +++
 
-[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/master/docs-chef-io/content/aws_elasticache_cluster.md)
+[\[edit on GitHub\]](https://github.com/inspec/inspec/blob/main/docs-chef-io/content/aws_elasticache_cluster.md)
```
Comment (IanMadd) [inline docs-chef-io/content/inspec/resources/aws_elasticache_cluster.md]: And one more.
```suggestion
```

**Comment by IanMadd** [comment]
Looks good aside from the suggestions.

**Review by IanMadd — APPROVED** [review]
LGTM

**Pull request merged**